tests/pages/base.py

```diff
@@ -33,6 +37,47 @@ def newsletter(self):
     def mozilla_newsletter(self):
         return MozillaNewsletterEmbedForm(self.base_url, self.selenium)
 
+    class Navigation(PageRegion):
+
+        _root_locator = (By.ID, 'nav-main')
+        _toggle_locator = (By.CLASS_NAME, 'toggle')
+        _menu_locator = (By.ID, 'nav-main-menu')
+        _about_locator = (By.CSS_SELECTOR, 'a[data-link-type="about"]')
+        _participate_locator = (By.CSS_SELECTOR, 'a[data-link-type="participate"]')
+        _firefox_locator = (By.CSS_SELECTOR, 'a[data-link-type="firefox"]')
+        _donate_locator = (By.CSS_SELECTOR, 'a[data-link-type="donate"]')
+
+        def show(self):
+            assert not self.is_displayed, 'Menu is already displayed'
+            self.find_element(self._toggle_locator).click()
+            self.wait.until(lambda s: self.is_displayed)
+            return self
+
+        @property
+        def is_displayed(self):
+            menu = self.find_element(self._menu_locator)
+            return menu.is_displayed() and menu.size['height'] == 177
```

Comment:
Ah, yes! I'd love to have a better indication that the menu is showing. I can't use `is_displayed` because then it tries clicking while it's still animating. I _could_ wait for the height to stop changing, but ideally I'd wait for something more concrete. Do you have any suggestions?